sql/information_schema/columns_table.go

```diff
@@ -167,14 +167,18 @@ func columnsRowIter(ctx *sql.Context, cat sql.Catalog, columnNameToDefault map[s
 					ordinalPos uint64
 					colDefault interface{}
 					charMaxLen interface{}
-					colType    string
+					columnKey  string
+					colType    = strings.ToLower(c.Type.String())
 				)
+
+				ordinalPos = uint64(i + 1)
 				if c.Nullable {
 					nullable = "YES"
 				} else {
 					nullable = "NO"
```

Comment:
(minor) you could consider initializing nullable to "NO" in the var block above, and then checking for c.Nullable. That would get rid of two more lines of code for you.